docs/index.adoc

```diff
@@ -32,6 +32,32 @@ without a third-party intermediary.
 
 == System Architecture
 
+=== Wallet States
+
+There are six states for a wallet:
+
+- `UNKNOWN`: The wallet is unknown to the Bridge.
+- `LIVE`: The wallet can sweep deposits and accept redemption requests.
+- `MOVING_FUNDS`: The wallet was deemed unhealthy and is expected to move its
+  outstanding funds to another wallet. The wallet can still fulfill its
+  pending redemption requests although new redemption requests and new deposit
+  reveals are not accepted.
+- `CLOSING`: The wallet moved or redeemed all its funds and is in the closing
+  period where it is still a subject of fraud challenges and must defend against
+  them. This state is needed to protect against deposit frauds on deposits
+  revealed but not swept.
+- `CLOSED`: The wallet finalized the closing period successfully and can no
+  longer perform any action in the Bridge.
+- `TERMINATED`: The wallet committed a fraud that was reported, did not move
+  funds to another wallet before a timeout, or did not sweep funds moved to it
+  from another wallet before a timeout. The wallet is blocked and can not
```

Comment:
> The wallet committed a fraud that was reported

The `notifyFraudChallengeDefeatTimeout` leading to TERMINATED state can be also called if fraud wasn't committed, but somebody accused wallet of committing it and the wallet did not defeat the challenge on time. I propose:

_The wallet did not move funds to another wallet before a timeout, did not sweep funds moved to it from another wallet before a timeout, or was accused of a fraud and did not defeat the fraud challenge before a timeout._

Reply:
Yeah, I agree it's an unlikely scenario. I thought that the description I proposed is closer to the actual implementation - but you're right, we don't need to match this description 1-to-1 with what's implemented, it's here more to give a general idea about the role of each state. I'm fine with leaving it as it is.